.github/workflows/release-branch.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Bump versions
       uses: eclipse-set/build/.github/actions/set-version@main
       with:
-        version: ${{ inputs.version }}.0-SNAPSHOT
+        version: ${{ inputs.version }}.1-SNAPSHOT
```

Comment:
What's the reason behind this change?

Reply:
I mixed up at which point we need to update the release branch development version. Reverted.